cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzurePrivateDnsZoneServiceEnum.java

```diff
@@ -3,14 +3,17 @@
 import static java.util.function.Function.identity;
 import static java.util.stream.Collectors.toMap;
 
+import java.util.List;
 import java.util.Map;
-import java.util.StringJoiner;
+import java.util.regex.Pattern;
 import java.util.stream.Stream;
 
-public enum AzurePrivateDnsZoneServiceEnum {
+public enum AzurePrivateDnsZoneServiceEnum implements AzurePrivateDnsZoneDescriptor {
 
-    POSTGRES("Microsoft.DBforPostgreSQL/servers", "postgresqlServer", "privatelink.postgres.database.azure.com", "postgres.database.azure.com"),
-    STORAGE("Microsoft.Storage/storageAccounts", "Blob", "privatelink.blob.core.windows.net", "blob.core.windows.net");
+    POSTGRES("Microsoft.DBforPostgreSQL/servers", "postgresqlServer", "privatelink.postgres.database.azure.com",
+            "postgres.database.azure.com", "privatelink\\.postgres\\.database\\.azure\\.com"),
+    STORAGE("Microsoft.Storage/storageAccounts", "Blob", "privatelink.blob.core.windows.net",
+            "blob.core.windows.net", "privatelink\\.blob\\.core\\.windows\\.net");
```

Comment:
could you point me to the unit/int tests where these patterns are tested?

Reply:
[AzurePrivateDnsZoneRegistrationEnumTest.java]

Reply:
Pattern tests were added.